deebot_client/map.py

```diff
@@ -194,15 +279,17 @@ def _update_trace_points(self, data: str) -> None:
         trace_points = _decompress_7z_base64_data(data)
 
         for i in range(0, len(trace_points), 5):
-            byte_position_x = struct.unpack("<h", trace_points[i : i + 2])
-            byte_position_y = struct.unpack("<h", trace_points[i + 2 : i + 4])
+            position_x: int = struct.unpack("<h", trace_points[i : i + 2])[0]
+            position_y: int = struct.unpack("<h", trace_points[i + 2 : i + 4])[0]
 
-            # Add To List
-            position_x = (int(byte_position_x[0] / 5)) + 400
-            position_y = (int(byte_position_y[0] / 5)) + 400
+            point_data = trace_points[i + 4]
 
-            self._map_data.trace_values.append(position_x)
-            self._map_data.trace_values.append(position_y)
+            connected = point_data >> 7 & 1 == 0
+            point_type = point_data & 1
```

Comment:
Can you please describe what the point_type is and where it will be used in the code?

Reply:
Currently... I have no idea. I had it in my old code, derived from the mobile app code of that time, when I extended the original Ecovacs Hass integration to have a map. However, trace points carry that information, but the code currently does not use them... Maybe the original app uses it to distinguish between vacuum and mop traces. I will try to figure out if there are differences between them.

Reply:
We should remove it for now. We can add it back later when we need it.

> extended the original Ecovacs Hass integration

Is your integration public available?

Reply:
No, but I can extract it to a public repo if interested.

Reply:
Code extracted here :) https://github.com/lukakama/hass-ecovacs-ext

Reply:
It looks like these are XML-based commands. Do you still have a robot that uses your component?

Reply:
Yes, but it is half-broken: it dies after a couple of minutes of cleaning, and the central rotating brush motor controller somewhat broke (it won't turn off even while charging... I had to disconnect the motor from the board). However it is usable for some tests.

Reply:
I remove the point type for now. If we need it later, we can always add it back